### PR TITLE
chore: updated apiUtils and getTracks file

### PR DESCRIPTION
**Pull request by rajat-ws**
## Description: 

- No change is UI 
- Updated the necessary files according to the template
- deleted non-used files

## Preview:

image

### CODE REVIEW

File: features/itunes/api/getTracks.ts [hunk 1]
```diff
@@ -1,5 +1,7 @@
-import { createApi, fetchBaseQuery } from "@reduxjs/toolkit/query/react";
-import { HYDRATE } from "next-redux-wrapper";
+// import { HYDRATE } from "next-redux-wrapper";
+// import { createApi, fetchBaseQuery } from "@reduxjs/toolkit/query/react";
```
Comment (praveenkumar1798) [inline features/itunes/api/getTracks.ts]: remove if we dont need it

File: pages/index.tsx [hunk 2]
```diff
@@ -2,18 +2,19 @@ import React, { memo } from "react";
...
+// import { getRecommendations } from "../services/root";
+
+// export async function getStaticProps() {
+//   const recommendations = await getRecommendations();
+//   return {
+//     props: {
+//       recommendations,
+//     },
+//   };
+// }
```
Comment (praveenkumar1798) [inline pages/index.tsx:14]: remove if not required

File: features/itunes/api/getTracks.ts (hunk 1 above)
Comment (rajat-ws) [reply features/itunes/api/getTracks.ts]: Removing it, updated accordingly.

File: pages/index.tsx (hunk 2 above)
Comment (rajat-ws) [reply pages/index.tsx:14]: It's going to be required in next PR

File: containers/Itunes/index.tsx [hunk 3]
```diff
@@ -62,8 +63,8 @@ export const TracksContainer: React.FC<ItunesContainerProps> = ({ recommendation
     <>
       <Container>
         <StyledRecommnendationWrapper>
-          <T id="recommended" styles={fonts.style.subheading()} />
-          <Recommended recommendations={recommendations} />
+          <T id="recommended" />
+          <Recommended />
```
Comment (praveenkumar1798) [inline containers/Itunes/index.tsx:67]: But how are we passing the recommendations data to this component ?

File: services/root.js [hunk 4]
```diff
@@ -0,0 +1,7 @@
+import { generateApiClient } from "@app/utils/apiUtils";
+const itunesApi = generateApiClient("itunes");
+
+export const getRecommendations = async () => {
+  const res = await itunesApi.get(`search?term=`);
+  console.log("res", res);
+};
```
Comment (praveenkumar1798) [inline services/root.js:7]: You aren't returning anything here

File: containers/Itunes/index.tsx (hunk 3 above)
Comment (rajat-ws) [reply containers/Itunes/index.tsx:67]: We aren't passing any props for now. It's just the UI for now.

File: services/root.js (hunk 4 above)
Comment (rajat-ws) [reply services/root.js:7]: Working on this in next PR

**Pull request merged**